### DISCUSSION TITLE
MAC OS kitty terminal screen flicker

### DISCUSSION THREAD

**Discussion by silicali** [kovidgoyal/kitty#4346] [General]
I'm using latest nightly build and past one week I noticed kitty terminal screen is flickering while running on battery.
If I disable "Automatic Graphic Switching" in battery settings, then it goes away.

Any suggestion!!?

kitty 0.23.1 created by Kovid Goyal
Darwin user-mbp 20.6.0 Darwin Kernel Version 20.6.0: Tue Oct 12 18:33:42 PDT 2021; root:xnu-7195.141.8~1/RELEASE_X86_64 x86_64
ProductName:    macOS ProductVersion:   11.6.1 BuildVersion:    20G224
Frozen: True
Paths:
  kitty: /Applications/kitty.app/Contents/MacOS/kitty
  base dir: /Applications/kitty.app/Contents/Resources/kitty
  extensions dir: /Applications/kitty.app/Contents/Resources/Python/lib/kitty-extensions
  system shell: /usr/local/bin/bash
Loaded config files:
  /Users/user-mbp/Desktop/.config/kitty/kitty.conf

Config options different from defaults:
allow_remote_control      y
bell_on_tab               False
bold_font                 JetBrains Mono Bold
bold_italic_font          IBM Plex Mono Bold Italic
clipboard_control         ('write-clipboard', 'write-primary', 'no-append')
clipboard_max_size        10.0
confirm_os_window_close   1
copy_on_select            clipboard
editor                    /usr/local/bin/nvim
enable_audio_bell         False
env:
{'LC_THEME': 'light'}
focus_follows_mouse       True
font_family               JetBrains Mono Medium
font_size                 15.0
hide_window_decorations   2
initial_window_height     (860, 'px')
initial_window_width      (1280, 'px')
italic_font               IBM Plex Mono Medium Italic
macos_option_as_alt       3
macos_thicken_font        0.1
scrollback_lines          50000
select_by_word_characters :@-./_~?&=%+#
shell                     /usr/local/bin/bash
tab_bar_edge              1
tab_bar_style             powerline
tab_powerline_style       round
url_prefixes              ('http', 'https', 'file', 'ftp', 'irc', 'gopher', 'mailto', 'news', 'git')
url_style                 1
window_margin_width       FloatEdges(left=2.0, top=2.0, right=0, bottom=0)
window_padding_width      FloatEdges(left=2.0, top=2.0, right=0, bottom=0)
:

**Comment by kovidgoyal** [comment]
Nothing has changed in how kitty draws the screen as far as I know, but feel free to
bisect to see if you can find a problematic commit.

**Reply by silicali** [reply]
may not be related to kitty terminal. so closing it
